README.md

```diff
@@ -353,6 +353,64 @@ Data:
 }
 ```
 
+### isValidIdentifier
```

Comment:
I think this keyword is missing from the index of the markdown file, as well as the header for section ``Extended keywords for ontology and taxonomy validation`` :)